### PR TITLE
Figs for lasso expes

### PR DESCRIPTION
**Pull request by QB3**

### CODE REVIEW

**Comment by QB3** [comment]
LGTM @agramfort, wdyt?

File: sparse_ho/forward.py [hunk 1]
```diff
@@ -130,10 +130,10 @@ def get_beta_jac_iterdiff(
                 X, y, beta, dbeta, r, dr, alphas, L, compute_jac=compute_jac)
 
         pobj.append(model._get_pobj(r, beta, alphas, y))
-        print(pobj[-1])
+        # print(pobj[-1])
```
Comment (agramfort) [inline sparse_ho/forward.py:133]: cleanup?

File: sparse_ho/forward.py [hunk 2]
```diff
@@ -130,10 +130,10 @@ def get_beta_jac_iterdiff(
...
 
         if i > 1:
-            assert pobj[-1] - pobj[-2] <= 1e-2 * np.abs(pobj[0])
+            # assert pobj[-1] - pobj[-2] <= 1e-2 * np.abs(pobj[0])
```
Comment (agramfort) [inline sparse_ho/forward.py:136]: cleanup?

File: sparse_ho/grid_search.py [hunk 3]
```diff
@@ -59,14 +59,12 @@ def grid_search(
             log_alpha = log_alphas[i]
         if samp == "lhs":
             log_alpha = log_alpha[0]
-        g_func = algo.criterion.get_val(
-            log_alpha, tol=tol)
+        # g_func = algo.criterion.get_val(
+        #     log_alpha, tol=tol)
```
Comment (agramfort) [inline sparse_ho/grid_search.py:63]: cleanup?

**Pull request merged**